### ISSUE TITLE
OhioStateNavbar - Support custom skip link targets

### ISSUE THREAD

**Issue by McManning** [osuresearch/ui#40]
The navbar/heading component for OSU needs to be able to target a specific ID rather than a hardcoded value for the skip link.

Use case:
- Integration with frameworks like Docusaurus that have their own explicit skip link target ID and cannot be changed to ours.

**Comment by McManning** [comment]
Dropping this. Adding skip links isn't the responsibility of this component. One can just be added before this element in the DOM.
